### PR TITLE
Update edit project page to cope with library barcode changes

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: miso-web/src/main/webapp/pages/editProject.jsp [hunk 1]
```diff
@@ -1113,11 +1111,9 @@
...
                   <td><c:if test="${not empty grouplib.tagBarcodes}">
-                    <c:forEach items="${grouplib.tagBarcodes}" varStatus="status" var="barcodemap">
-                      ${status.count}: ${barcodemap.value.name} (${barcodemap.value.sequence})
-                      <c:if test="${status.count lt fn:length(grouplib.tagBarcodes)}">
-                        <br/>
-                      </c:if>
+                    <c:forEach items="${grouplib.tagBarcodes}" varStatus="status" var="barcode">
+                       <c:if test="${status.index gt 0}"><br/></c:if>
+                       ${status.count}: ${barcod.name} (${barcode.sequence})
                     </c:forEach>
```
Comment (callunity) [inline miso-web/src/main/webapp/pages/editProject.jsp]: `${barcod.name}` <- missing a terminal 'e'

**Comment by callunity** [comment]
:+1: after typo fix

File: miso-web/src/main/webapp/pages/editProject.jsp (hunk 1 above)
Comment (apmasell) [inline miso-web/src/main/webapp/pages/editProject.jsp]: Fixed. Thanks.

**Comment by tonydebat** [comment]
👍

**Pull request merged**